Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/powerup2018/StatsEngine.java

```diff
@@ -656,18 +658,36 @@ private void fillInTeleopStats(TeamStatsReport teamStatsReport) {
 
                     if(inSwitchCycle) {
                         teamStatsReport.switchAvgCycleTime += cycleTime;
+                        teamStatsReport.switchCycleTimes.add(cycleTime);
                         cyclesInThisMatch.cycles.add(cubeCycle.clone(Cycle.CycleType.SWITCH));
                         inSwitchCycle = false;
+
+                        if(match.preGameObject.matchNumber > LATER_MATCH_START) {
```

Comment:
If matchNumber > 35 ?
Can you explain what's going on here?

Reply:
OOHH, is this a "are they getting better over time" thing?

Reply:
It is, though for some reason in the data for ryerson setting it past about 40 basically makes all the stats wrong and even at 35 some of them look very wonky